Automattic/jetpack
# Jetpack: move VideoPress block to extended blocks folder

The current implementation of VideoPress is not a block by itself but it extends and also overwrites the core/video block.
Thus, as an initial step to work on the new VideoPress block, it makes sense to move the code from extensions/blocks/videopress to extensions/extended-block/core-video folder to pave the way for adding, in follow-up tasks, the new block.

Fixes #

#### Changes proposed in this Pull Request:

* Jetpack: move VideoPress block to extended blocks folder

#### Other information:


#### Jetpack product discussion



No

#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






It doesn't change anything in terms of the feature implementation, so the testing instructions are limited to verifying that the current implementation of VideoPress still works without any changes.

[Jetapack videopress product page](https://jetpack.com/support/jetpack-videopress/add-video-block-editor/)

* Use a testing site with Jetpack with VideoPress feature supported. You may need to buy a plan/product
* Add a `core/video` block
* Upload a video
* Confirm you see the file uploading with the VideoPress feature
* Change the block properties using the toolbar and/or block inspector
* Try to test as much as possible all features confirming it works as expected.
* Test the feature in the block editor context as well as in the front-end

core editor context | frontend
------|------
<img width="704" alt="image" src="https://user-images.githubusercontent.com/77539/175121058-454e6f01-49b9-4403-a0d0-6e76822b0aa2.png">|<img width="705" alt="image" src="https://user-images.githubusercontent.com/77539/175121088-6e5d2fc4-e7f6-44f3-83ab-e84ce5337378.png">




---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - 0-as-1202490094773346
